examples/from_source/package.json

```diff
@@ -3,7 +3,8 @@
   "private": true,
   "devDependencies": {
     "@types/node": "12.12.50",
-    "protobufjs": "6.10.1",
+    "google-protobuf": "^3.6.1",
```

Comment:
if the example has to change, that means it's a breaking change...

Reply:
I have no clue how I can make this non-breaking change... may be you have something on your mind.

Reply:
also, I have no idea why this was needed in the first place. Shouldn’t rules_nodejs list this as a dependency even if the repository is consumed from the source? 

I only added this because the tests were yelling at me

I believe that the problem lies somewhere else.

Reply:
Shouldn’t this be handled by rules_nodejs_dev_dependencies? perhaps something similar? as I don’t rely on the consumer's npm repository, I don’t see why this should be listed as a dependency by the consumer.

Reply:
rules_nodejs_dev_dependencies is only for when you want to consume our rules from source. normal users shouldn't need it. but since this is the examples/from_source it actually does make sense here

Reply:
should I add it to this example? or shall we make separate PR for this?

Reply:
sorry it was my mistake, as it's building rules_nodejs from source it's expected that our dev dependencies leak through. So far we have the rules_nodejs_dev_dependencies for *starlark* dependencies, but we don't have anything for *npm* dependencies (in theory it should be the root package json but we don't actually publish `@bazel/javascript` we treat it as private)

Reply:
Hmm. That’s what I was expecting. so is this okay to have this change?